test_data/std/fingerprint_test.asm

```diff
@@ -0,0 +1,41 @@
+use std::math::fp2::from_base;
+use std::math::fp2::Fp2;
+use std::math::fp2::eval_ext;
+use std::math::fp2::unpack_ext_array;
+use std::math::fp2::constrain_eq_ext;
+use std::prover::challenge;
+use std::protocols::fingerprint::fingerprint;
+use std::protocols::fingerprint::fingerprint_inter;
+use std::array;
+use std::convert::expr;
+use std::prover::eval;
+
+machine Main with degree: 2048 {
+
+    col witness x(i) query Query::Hint(42);
+
+    // Fold tuple [x, x + 1, ..., x + n - 1]
+    // Note that setting a fairly large `n` tests that performance is *not* exponential in `n`.
```

Comment:
```suggestion
    // Note by setting a fairly large `n` tests that performance is *not* exponential in `n`.
```

?

Reply:
I think it's correct as is? 😄 
Or it could be
"Note that, by setting a fairly large `n`, we test that performance is *not* exponential in `n`."

Reply:
Went wit that!